qt/TorrentDelegate.cc

```diff
@@ -194,13 +194,15 @@ QString TorrentDelegate::progressString(Torrent const& tor)
             //: %2 is the torrent's total size,
```

Comment:
A nice, generic mechanism for this would be nice but it's been 5 months since I made that comment and nothing's happened on that front, so let's not block this feature on it.